.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint Python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit black codespell flake8 isort mypy pyupgrade safety
+      - run: bandit --recursive --skip B101,B108,B110,B310,B311,B403,B404,B603 .
+      - run: black --check . || true
+      - run: codespell || true  # --ignore-words-list="ans,kno,lof,te,thisy" --skip="*.css,*.js,*.lock"
+      - run: flake8 . --count --select=E501,E9,F63,F7,F82 --max-complexity=18
```

Comment:
- Limit the scope to `python/`, `max-line-length=80` (since we're using the google style)
- Also, why do we have two `flake8`s?

Reply:
```
./python/open3d/visualization/tensorboard_plugin/plugin_data_pb2.py:45:216: E501 line too long 2444 characters)
```

The `--select=E501,E9,F63,F7,F82` run focuses on _show-stopper_ issues that effect program correctness and runtime safety.

The `--exit-zero` run is designed to show us ___all___ issues (some of which we may not choose to fix).